service/src/main/kotlin/io/provenance/api/domain/usecase/cee/approve/ApproveContractBatchExecution.kt

```diff
@@ -53,19 +58,34 @@ class ApproveContractBatchExecution(
 
                     it.forEach { executions ->
                         client.respondWithApproval(executions.first, broadcast.txhash)
-                        responses.add(ApproveContractExecutionResponse(Base64.getEncoder().encodeToString(executions.first.toByteArray()), broadcast.toTxResponse()))
+                        responses.add(
+                            ApproveContractExecutionResponse(
+                                Base64.getEncoder().encodeToString(executions.first.toByteArray()),
+                                broadcast.toTxResponse(),
+                                it.map { UUID.nameUUIDFromBytes(it.first.result.scopeOrNull()?.scope?.scope?.scopeId?.toByteArray()) }
```

Comment:
I'm unfamiliar with nameUUIDFromBytes so I'll look that up, but the UUID should be in the scopeIdInfo like so:
`it.map { UUID.fromString(it.first.result.scopeOrNull()?.scope?.scopeIdInfo?.scopeUuid) }`

Reply:
changed to use `.fromString`